packages/app/src/cli/services/build/extension.ts

```diff
@@ -141,11 +142,18 @@ export async function buildFunctionExtension(
   }
 
   try {
+    const bundlePath = joinPath(extension.outputPath.split('/').slice(0, -2).join('/'), 'index.wasm')
+    extension.outputPath = joinPath(extension.directory, joinPath('dist', 'function.wasm'))
     if (extension.isJavaScript) {
       await runCommandOrBuildJSFunction(extension, options)
     } else {
       await buildOtherFunction(extension, options)
     }
+    if (fileExistsSync(extension.outputPath)) {
+      const base64Contents = await readFile(extension.outputPath, {encoding: 'base64'})
+      await touchFile(bundlePath)
+      await writeFile(bundlePath, base64Contents)
```

Comment:
Yeah thats how I had it before but I had to change that to base64 encode this because I had to base64 encode the wasm file that I'm sending over the network for draftable dev (utf-8 encoding doesn't work for wasm files) and so the code obtaining the file in Core now expects it to be bae64 encoded.

Reply:
Yes right, exactly, so then we can go back and simplify this.